### PR TITLE
Add Slayers Guild token SL4Y to VeChain token registry

### PR DESCRIPTION
**Pull request by mrwhittleman**
Adding Slayers Guild project token SL4Y to VeChain token registry.

### CODE REVIEW

**Comment by darrenvechain** [comment]
@mrwhittleman 0x4B85757Bcf693f742003f2D5529CDc1672392F16 is not a contract on testnet. It looks like you copied and pasted from mainnet.

You can either deploy a contract and update the address, or remove the testnet contract

**Comment by mrwhittleman** [comment]
> @mrwhittleman 0x4B85757Bcf693f742003f2D5529CDc1672392F16 is not a contract on testnet. It looks like you copied and pasted from mainnet.
> 
> You can either deploy a contract and update the address, or remove the testnet contract

I will remove the testnet contract!

**Comment by mrwhittleman** [comment]
@darrenvechain All good?

**Comment by darrenvechain** [comment]
@mrwhittleman I still see the testnet contract 
image

**Comment by mrwhittleman** [comment]
> @mrwhittleman I still see the testnet contract image

@darrenvechain Sorry about that. I thought I had deleted it before...

**Comment by mrwhittleman** [comment]
@darrenvechain - Can you review and approve? Or let me know of any outstanding issues? Thank you!

File: tokens/main/0x4b85757bcf693f742003f2d5529cdc1672392f16/additional.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "website":"https://www.slayersguild.io/",
```
[the remark this answers is not in the document]
Comment (mrwhittleman) [reply tokens/main/0x4b85757bcf693f742003f2d5529cdc1672392f16/additional.json:2]: We are working on the dApp currently. Should be up in a couple weeks. Our gitbook is already up https://docs.slayersguild.io

[the remark this answers is not in the document]
Comment (mrwhittleman) [reply tokens/main/0x4b85757bcf693f742003f2d5529cdc1672392f16/additional.json:2]: If that is unacceptable I can change the website to the docs.

**Comment by mrwhittleman** [comment]
@libotony @darrenvechain updated additional .json to have website reflect our gitbook until dApp is up.

File: tokens/main/0x4b85757bcf693f742003f2d5529cdc1672392f16/additional.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "links":{
+        "twitter":"https://twitter.com/vechainslayers",
```
Comment (libotony) [inline tokens/main/0x4b85757bcf693f742003f2d5529cdc1672392f16/additional.json:3]:
```suggestion
        "twitter":"https://twitter.com/vechainslayers"
```

**Review by libotony — CHANGES_REQUESTED** [review]

**Review by libotony — APPROVED** [review]

**Pull request merged**